redi-connect-front/src/components/organisms/RediHero.tsx

```diff
@@ -20,18 +15,20 @@ const RediHero = () => {
       <Container>
         <Columns vCentered>
           <Columns.Column size={5}>
-            <Heading>
-              {t('loggedOutArea.homePage.hero.headline')}
+            <Heading className="five-bs">
```

Comment:
Please don't define a global class for one time use. Just create a local class for the component.

Reply:
outsourced to `RediHero.scss`